### ISSUE TITLE
Getting the flags from the manual

### ISSUE THREAD

**Issue by zerothi** [siesta-project/siesta-vim#4]
Since 4.1r765 the manual writes out to a single file all fdf-flags defined. 

@arthurfl 
Would it make sense to read this in a stringent manner to reduce the cluttering of the `siesta.vim` file?

I.e. I would like feedback on this possibility and how to do this easily in vim rc files?

**Comment by arthurfl** [comment]
hi @zerothi , 

That's a very nice feature - I'll give it some thoughts. Treating fdf-flags in a more systematic way would be great, especially for updating. One would need to find a way to broadcast the content of these fdf files into `siesta.vim`. 

I can see two limitations: 

- in `siesta.vim`, fdf-flags are broken down into two subcategories, general (`siestaKeys`) and output (`siestaOutput`); 
- the regex is not always just the fdf-flag with the allowed `._-` insertions (see eg. `DMInitBulk`): all keywords cannot be treated exactly in the same way, and it doesn't seem possible to avoid exceptions.

**Comment by zerothi** [comment]
@arthurfl no, I agree. However, just having the plain fdf-flags may provide a substantial fraction of what is necessary?
I can readily extend the .tex document to write-out a list of option names. If this could also provide some more detail?
